### DISCUSSION TITLE
Adding more spacecrafts and produce GIF animation

### DISCUSSION THREAD

**Discussion by MohamedNedal** [jgieseler/solarmach#14] [Q&A]
Hello, I wonder if it's possible to add more spacecraft, such as ACE and Wind satellites? 
Also, Would it be possible to generate a GIF animation within a specific period? 
Thanks! 
Mohamed

**Comment by jgieseler** [comment]
The mentioned ACE and Wind I usually omit because they would overlap with `'Earth'`. But you can already now replace `'Earth'` with `'Wind'`, which then will be plotted instead (doesn't work in the web-app, though).

Cf. also #8

**Comment by jgieseler** [comment]
About the GIF animation, one could quite easily achieve that by modifying my example from the [PyHC summer school notebook](https://github.com/heliophysicsPy/summer-school/blob/main/solarmach-tutorial/notebook.ipynb) (under _5.1 Loop over multiple datetimes (plots)_). 

One could of course write this as a new function of solarmach. But because this is quite a specific task that already has some kind of workaround, I think for me it has a rather low priority. But that shouldn't stop anyone else to write such a short function! 😉 I'd be glad to help!

**Comment by MohamedNedal** [comment]
I've checked the example in the summer school notebook, and for some reason the _imageio_ doesn't work. I'll try to see alternative options. Many thanks! :)
